core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -25,15 +26,13 @@ object StaticFile {
 
   val DefaultBufferSize = 10240
 
-  def fromString[F[_]: Sync: ContextShift](
+  def fromString[F[_]: Files: Sync](
```

Comment:
you don't want `Files: Sync`. It it works with just `Files` plus non effect capturing classes (e.g. `Monad` or `Concurrent`), then great, if not `Sync` is enough.